src/util/types.ts

```diff
@@ -830,6 +830,14 @@ const SVGElementPropKeys = [
   'angle',
 ];
 
+export type FilteredSvgElementType = 'svg' | 'polyline' | 'polygon';
+
+export const FilteredElementKeyMap: Record<FilteredSvgElementType, string[]> = {
```

Comment:
Could you please add a small comment? Is this a list of properties not to include per svg element type?